mkdocs.yml

```diff
@@ -3,7 +3,7 @@ repo_url: https://github.com/alphagov/di-identity-vocab
 theme:
     name: null
     custom_dir: 'mkdocs-govuk/govuk_theme'
-tech_docs:
```

Comment:
So I think you need this providing you are on the correct branch of mkdocs-govuk - match-techdocs-template, if you are on this branch the template files refer to config.tech_docs throughout and if you don't have it the formatting doesn't work.  Looks like it may have been a problem with pulling the submodules down at the right branch.